scripts/install.sh

```diff
@@ -52,6 +53,7 @@ main() {
       '--help')           set -- "$@" '-h'   ;;
       '--no-modify-path') set -- "$@" '-p'   ;;
       '--version')        set -- "$@" '-v'   ;;
+      '--init')           set -- "$@" '-i'   ;;
```

Comment:
Should this go into the installer? 🤔

Reply:
I don't think so, we shouldn't touch this script in this PR